Added a section about the env error log (#2838)
umbraco/UmbracoDocs: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md

From [Settings](../../Set-up/project-settings) - in the top-right corner - you'll find a lot more options to configure your project.

## Environment error log

> Each environment has an error log that appears only if you have any unread errors for that specific environment. You can view them by clicking on "View errors" in the environment menu.

Reviewer: ```suggestion
Each environment has an error log that appears only if you have any unread errors on that specific environment. You can view the errors by clicking on "View errors" in the environment menu.
```

> Once you're there you can manually mark each error as read which will move it from the "New" section to the "Read" section. Errors marked as read will be permanently deleted after 30 days.

Reviewer: ```suggestion
Once you're there, you can manually mark each error as read which will move it from the "New" section to the "Read" section. Errors marked as read will be permanently deleted after 30 days.
```

> During development, you can happen to gather a quite impressive number of errors which might cause the error page to load very slowly. A simple fix for that would be to locally connect to the database for that specific environment and delete those errors. You can read more about connecting to the environment database locally [here](../../Databases/Cloud-Database).

Reviewer: ```suggestion
During development, you can happen to gather a large number of errors which might cause the error page to load very slowly. A fix for that would be to locally connect to the database for that specific environment and delete the errors. You can read more about connecting to the environment database locally in the [section about databases on Umbraco Cloud](../../Databases/Cloud-Database).
```

> Environment errors are stored in the UCErrorLog table.

Reviewer: ```suggestion
Environment errors are stored in the `UCErrorLog` table.
```

```
DELETE TOP(90) PERCENT
  FROM [dbo].[UCErrorLog]
  WHERE [Read] = 0
```

> The query above will delete 90% of errors. It always begins deleting the oldest errors first. You can tweak the query to delete any % of errors you like by changing the number in the first row.

Reviewer: ```suggestion
The query above will delete 90% of errors. The query will always delete the oldest errors first. You can tweak the query to delete any percentage of errors by changing the number in the first row.
```
